projects/ancestor/util.py

```diff
@@ -0,0 +1,26 @@
+class Queue():
+    def __init__(self):
+        self.queue = []
+    def enqueue(self, value):
+        self.queue.append(value)
+    def dequeue(self):
+        if self.size() > 0:
+            return self.queue.pop(0)
+        else:
+            return None
+    def size(self):
+        return len(self.queue)
+
```

Comment:
Nice use of the util.py file to hold the Queue and Stack classes for your searches/traversals.